pkg/clientset/tiltapi/wire.go

```diff
@@ -0,0 +1,19 @@
+package tiltapi
```

Comment:
i'm a little nervous about mixing non-generated code with generated packages

Reply:
Agreed - thankfully this can actually just ago away entirely now in this case (see previous comment)